Maincpp.cpp

```diff
@@ -0,0 +1,48 @@
+#include <iostream>
+#include "Sorting.h"
+using namespace std;
+void printArr(double *arr, int length)
+{
+	for (int i = 0; i < length; i++)
+	{
+		cout << arr[i] << endl;
+	}
+}
+int main()
+{
+	{
+		Sorting s;
+
+		double* arr0 = new double[3];
+		arr0[0] = 9;
+		arr0[1] = 89;
+		arr0[2] = 10.4;
+
+		double* arr1 = new double[3];
+		arr1[0] = 9;
```

Comment:
nie dość, że wybrałeś tragiczny sposób definicji tablicy to jeszcze definiujesz taką samą 4 razy? poczytaj o przekazywaniu przez wartość i przez referencję. Przyda Ci się bardzo.